### ISSUE TITLE
question about binary classification

### ISSUE THREAD

**Issue by swangb** [vlfeat/matconvnet#87]
Hi,
I meet some problems when doing binary classification. I start with the pretrained imagenet-caffe-ref model and use two-class image data (face and background with label 1,2) 880(440 440) or 1760(440 1320) to fine-tune the model. I did the modification to the fc7 and also errorType as #55. I also change the learning ratio to be 10 times of origin for fc7. After the code run a few epoch, the object result decrease reasonably, however the error rate always converges to the ratio of data: for 880 data the error rate is 50% and for 1760 data the error is 25%. Is this the bug of the binary errorType evaluation?
Thank you!

**Comment by swangb** [comment]
Hi, did any one have the same problem?

**Comment by xcyan** [comment]
It seems the learning got stuck at local optimal. First, you may want to
initialize fc7 differently. Alternatively, you can tune the learning rate
or use advanced optimization methods like cg, nestrov momentum, or rmsprop.

**Comment by swangb** [comment]
Hi Xinchen, thanks for your comments! 
A successful example of fine-tuning of imagennet caffe model is recognizing the image style paper. But I find that the images used to fine-tune the model is kind of similar to the imagenet data. However the data I used is grayscale and low resolution. I think this might be the problem. Maybe the imagenet model is not useful for my data and not possible to be fine-tuned to help to learn discriminant representation for my data (I re-initialize the last 4 layer with very high learning ratio and find the error ratio is still converge to the data). Do you agree with me?

**Comment by xcyan** [comment]
You may want to build your own model and see what will happen.
